packages/search/tsconfig.json

```diff
@@ -5,5 +5,6 @@
 		"outDir": "dist/esm",
 		"rootDir": "src"
 	},
-	"include": [ "src" ]
+	"include": [ "src" ],
+	"exclude": [ "**/test/*" ]
```

Comment:
I'm getting type errors in the tests file for missing definitions for `describe` and `it`, which I'm guessing is why you excluded the tests here?

I don't fully understand why some Jest files seem to get the `describe` definitions automatically and some don't, but I tried fixing this locally by adding `types: [ "jest" ]` to the `compilerOptions` and it seemed to do the trick.